breaking-front/src/pages/PostWrite/units/PostWriteUploadMediaForm.js

```diff
@@ -1,83 +1,105 @@
           <Style.Carousel ref={caruselRef}>
-            <Style.UploadPreviewImage>
-              <Style.UploadBox></Style.UploadBox>
-              <Style.XMarkIconContainer>
-                <XMarkIcon />
-              </Style.XMarkIconContainer>
-            </Style.UploadPreviewImage>
-
-            <Style.UploadPreviewImage>
-              <Style.UploadBox></Style.UploadBox>
-              <Style.XMarkIconContainer>
-                <XMarkIcon />
-              </Style.XMarkIconContainer>
-            </Style.UploadPreviewImage>
-
-            <Style.UploadPreviewImage>
-              <Style.UploadBox></Style.UploadBox>
-              <Style.XMarkIconContainer>
-                <XMarkIcon />
-              </Style.XMarkIconContainer>
-            </Style.UploadPreviewImage>
-
-            <Style.UploadPreviewImage>
-              <Style.UploadBox></Style.UploadBox>
-              <Style.XMarkIconContainer>
-                <XMarkIcon />
-              </Style.XMarkIconContainer>
-            </Style.UploadPreviewImage>
-
-            <Style.UploadPreviewImage>
-              <Style.UploadBox></Style.UploadBox>
-              <Style.XMarkIconContainer>
-                <XMarkIcon />
-              </Style.XMarkIconContainer>
-            </Style.UploadPreviewImage>
-
-            <Style.UploadPreviewImage>
-              <Style.UploadBox></Style.UploadBox>
-              <Style.XMarkIconContainer>
-                <XMarkIcon />
-              </Style.XMarkIconContainer>
-            </Style.UploadPreviewImage>
-
-            <Style.UploadPreviewImage>
-              <Style.UploadBox></Style.UploadBox>
-              <Style.XMarkIconContainer>
-                <XMarkIcon />
-              </Style.XMarkIconContainer>
-            </Style.UploadPreviewImage>
+            {filesThumbnail &&
+              filesThumbnail.map((file, index) => {
+                if (file.type === 'image') {
+                  return (
+                    <Style.PreviewImageContainer key={index}>
+                      <Style.PreviewImage src={file.url} />
+                      <Style.XMarkIconContainer
+                        onClick={() => deleteFile(index)}
+                      >
+                        <XMarkIcon />
+                      </Style.XMarkIconContainer>
+                    </Style.PreviewImageContainer>
+                  );
+                } else if (file.type === 'video') {
+                  return (
+                    <Style.PreviewImageContainer key={index}>
+                      <Style.PreviewVedio src={file.url} />
+                      <Style.XMarkIconContainer
+                        onClick={() => deleteFile(index)}
+                      >
+                        <XMarkIcon />
+                      </Style.XMarkIconContainer>
+                    </Style.PreviewImageContainer>
+                  );
+                } else {
+                  return <></>;
+                }
+              })}
```

Comment:
```suggestion
            {filesThumbnail &&
              filesThumbnail.map((file, index) => (
                <Style.PreviewImageContainer key={index}>
                  {file.type === 'image' ? (
                    <Style.PreviewImage src={file.url} />
                  ) : (
                    <Style.PreviewVideo src={file.url} />
                  )}
                  <Style.XMarkIconContainer onClick={() => deleteFile(index)}>
                    <XMarkIcon />
                  </Style.XMarkIconContainer>
                </Style.PreviewImageContainer>
              ))}
```
key값을 index로 넣으면 안좋다고 합니다.
https://medium.com/sjk5766/react-%EB%B0%B0%EC%97%B4%EC%9D%98-index%EB%A5%BC-key%EB%A1%9C-%EC%93%B0%EB%A9%B4-%EC%95%88%EB%90%98%EB%8A%94-%EC%9D%B4%EC%9C%A0-3ce48b3a18fb